### PR TITLE
Add bump-kubevirtci make target

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

Add bump-kubevirtci make target

```release-note
NONE
```

### CODE REVIEW

File: hack/bump-kubevirtci.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#/bin/bash
+#!/bin/bash
```
Comment (oshoval) [inline hack/bump-kubevirtci.sh]: /cc @danielBelenky

Comment (danielBelenky) [reply hack/bump-kubevirtci.sh]: :)

**Pull request merged**